pkg/internal/cni-conf-template.json

```diff
@@ -11,23 +13,36 @@
 	},
 	"Policies":[
 		{
-			"Name":"EndpointPolicy",
-			"Value":{
-				"Type":"OutBoundNAT",
-				"ExceptionList":[
-					"service_network_cidr"
-				],
-				"DestinationPrefix":"",
-				"NeedEncap":false
+			"Name": "EndpointPolicy",
+			"Value": {
+				"Type": "OutBoundNAT",
+				"Settings": {
+					"ExceptionList": [
+						"service_network_cidr"
+					],
+					"DestinationPrefix": "",
+					"NeedEncap": false
+				}
 			}
 		},
 		{
-			"Name":"EndpointPolicy",
-			"Value":{
-				"Type":"ROUTE",
-				"ExceptionList":[],
-				"DestinationPrefix":"service_network_cidr",
-				"NeedEncap":true
+			"Name": "EndpointPolicy",
+			"Value": {
+				"Type": "SDNROUTE",
+				"Settings": {
+					"ExceptionList": [],
+					"DestinationPrefix": "service_network_cidr",
+					"NeedEncap": true
+				}
+			}
+		},
+		{
+			"Name": "EndpointPolicy",
+			"Value": {
+				"Type": "ProviderAddress",
+				"Settings": {
+					"ProviderAddress": "ip_host"
+				}
```

Comment:
Are these changes compatible with dockershim?
If you are leaving the `dockerRuntime` flag within the code we have to test both paths, dockershim and containerd.

From our discussion, if we are targeting 6.0.0 which is incompatible with dockershim, the `dockerRuntime` flag should be removed, as by leaving it in we are include a broken code path.

Reply:
dockerRuntime flag usage has been removed in first commit.